### DISCUSSION TITLE
Return old logs and output buttons UI

### DISCUSSION THREAD

**Discussion by busyscout** [dbeaver/dbeaver#11501] [Ideas]
Since version 7.3.3 buttons for output and logs in sql editor have text labels.
It would be nice to have an option to remove text from these button (and make them look as they were before).
Reason:
1) they take more vertical space now
2) they take more attention now and distract from query text
3) they differ from sibling buttons in the same panel
![2021-02-19 19_29_38-DBeaver 7 3 5 - _none_ Script-204](https://user-images.githubusercontent.com/23096209/108532837-57263400-72e9-11eb-80d0-1e040148d113.png)

**Comment by kseniiaguzeeva** [comment]
I agree with you. From the other side, small icons are hard to see. I will convert the ticket into discussion and let's see the demand for this task.
